test/utils.js

```diff
@@ -27,11 +27,17 @@ function getBitwise(value) {
  *     false: The distance between a and b is far away from the given ULP distance.
  */
 assert.isAlmostEqualUlp = function(a, b, nulp, message) {
-  const aBitwise = getBitwise(a);
-  const bBitwise = getBitwise(b);
-  let distance = aBitwise - bBitwise;
-  distance = distance >= 0 ? distance : -distance;
-  return assert.isTrue(distance <= nulp, message);
+  if (typeof(a) == 'number') {
+    const aBitwise = getBitwise(a);
+    const bBitwise = getBitwise(b);
+    let distance = aBitwise - bBitwise;
+    distance = distance >= 0 ? distance : -distance;
+    return assert.isTrue(distance <= nulp, message);
+  } else {
+    let distance = a-b;
```

Comment:
[nit] `a - b` consistent spacing with line 33.

Reply:
(interesting - so in Javascript, int64 is considered not a "number" 🙃)

Reply:
The integer of type INT64 exceeds the range of JavaScript's Number type and belongs to the integer of BigInteger type. The JavaScript exception "can't convert BigInt to number" occurs when an arithmetic operation involves a mix of BigInt and Number values. FYI: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cant_convert_BigInt_to_number?retiredLocale=pl